distribution/smarthome/conf/things/demo.things

```diff
@@ -1,3 +1,3 @@
-yahooweather:weather:berlin [ location="638242" ]
-yahooweather:weather:losangeles "Los Angeles" [ location="2442047", unit="us", refresh=120 ]
+yahooweather:weather:berlin [ location=638242 ]
+yahooweather:weather:losangeles "Los Angeles" [ location=2442047, unit="us", refresh=120 ]
```

Comment:
They are related to the fact that the Yahoo binding is used as the reference implementation for this feature and as thus it has been cleaned up on the way. Ok for me, but if you prefer, we can also ask @tomhoefer to split it in separate commits.